airflow_clickhouse_plugin/hooks/clickhouse_hook.py

```diff
@@ -1,24 +1,27 @@
 from itertools import islice
 from typing import *
 
-from airflow.hooks.base_hook import BaseHook
+from airflow.hooks.dbapi import DbApiHook
 from clickhouse_driver import Client
 
 
-class ClickHouseHook(BaseHook):
-    DEFAULT_CONN_ID = 'clickhouse_default'
+class ClickHouseHook(DbApiHook):
+    conn_name_attr = 'clickhouse_conn_id'
```

Comment:
Is this used anywhere except `setattr`/`getattr` calls within `DbApiHook` methods?

Reply:
The DbApiHook needs this variable to work, it's used in all the implementations. If you don't put it the DbApiHook class triggers an exception in the constructor. https://github.com/apache/airflow/blob/a9ac2b040b64de1aa5d9c2b9def33334e36a8d22/airflow/hooks/dbapi.py#L60